stable/ambassador/ci/ci-values.yaml

```diff
@@ -0,0 +1,14 @@
+prometheusExporter:
+  enabled: true
+
+env:
+  AMBASSADOR_SINGLE_NAMESPACE: true
+
+volumes:
+  - name: nothing
+    emptyDir: {}
+
```

Comment:
Can you add daemonSet: true to this and create a default-values.yaml that's empty?